Layout/sidebar.css

```diff
@@ -2,10 +2,10 @@
 
 .sidebar {
 	width: 25%;
-	height: 82vh;
+	height: 102vh;
```

Comment:
try to avoid putting random numbers like 102, 19rem 1.6rem.. keep them std like 20, 1.5 etc

Reply:
That's a great point to add will keep in mind for sure✅